Android/Java/app/src/main/java/com/unity3d/levelplaydemo/DemoActivity.java

```diff
@@ -12,18 +12,15 @@
 import android.widget.FrameLayout;
 import android.widget.TextView;
 import androidx.multidex.BuildConfig;
-import com.ironsource.mediationsdk.IronSource;
-import com.ironsource.mediationsdk.integration.IntegrationHelper;
-import com.ironsource.mediationsdk.model.Placement;
-import com.ironsource.mediationsdk.utils.IronSourceUtils;
 import com.unity3d.mediation.LevelPlay;
```

Comment:
I would remove this comment, it's confusing
"// Unlike Rewarded Videos there are no placements"